src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -87,6 +87,7 @@ def execute(self, unexpanded_argv):
         if len(argv) == 0:
             az_subparser = self.parser.subparsers[tuple()]
             _help.show_welcome(az_subparser)
+            log_telemetry('welcome')
```

Comment:
yep, we want to see usage patterns and if people are relying on help or welcome for information frequently

---
In reply to: [79206649](https://github.com/Azure/azure-cli/pull/910#discussion_r79206649) [](ancestors = 79206649)